### ISSUE TITLE
[Toolkit bug]: RTX Remix toolkit crashes on startup

### ISSUE THREAD

**Issue by MartinPihrtJR** [NVIDIAGameWorks/rtx-remix#399]
### Describe the bug

While starting the app, the RTX Remix loading box pops up, then nothing happens next. (see video) Looking at the logs reveals that the app crashed. 

https://github.com/NVIDIAGameWorks/rtx-remix/assets/64539948/fc5d9c6f-696f-4987-9916-34925cdf144a

Looking at the log myself, most of the errors seem to be caused by 
**OSError: [WinError 87] The parameter is incorrect: 'C:'**
and later on by
**FileNotFoundError: [WinError 206] The filename or extension is too long: 'C:\\Program Files\\Oculus\\Support\\oculus-runtime'**
Log of crash:
[kit_20240202_131950.log](https://github.com/NVIDIAGameWorks/rtx-remix/files/14139993/kit_20240202_131950.log)

### How do you reproduce the bug?

1. Launch RTX Remix from NVIDIA Omniverse
2. RTX Remix crashes

### What is the expected behavior?

The toolkit should start up.

### Version

2024.1.1

**Comment by sambvfx** [comment]
Hey @MartinPihrtJR , thanks for reporting this. Something is definitely strange here. Are you still having this issue?

**Comment by MartinPihrtJR** [comment]
@sambvfx yes, the issue still persists. Tried launching it yesterday (after clean install), with no success. I also tried switching the GPU driver to studio, with no luck at all.

**Comment by KewkLW** [comment]
I believe I'm having the same issue. I'll start a new bug if needed but here's the rather huge log. 
[kit_20240213_154842.log](https://github.com/NVIDIAGameWorks/rtx-remix/files/14273727/kit_20240213_154842.log)

This is from a fresh install and trying to run.

**Comment by sambvfx** [comment]
Thanks for the logs, I've filed as REMIX-2793 internally for us to dive in and figure out what's going on.

**Comment by mbgamer05** [comment]
using the newest version of the toolkit getting the same issue, i don't know what's causing it and it's just quitting it's self without an error message of any kind

**Comment by MartinPihrtJR** [comment]
### UPDATE
After updating to version 2024.2.1 the issue persists. Application starts up to splashscreen, then crashes with the crashreporter running in the background (I don't even know if it sends the crashdump to nvidia servers).

Looking at the logs myslef, the main issue is still the **OSError: [WinError 87] The parameter is incorrect: 'C:'** error, while trying to load a python module called OmniGeospatialSchema. The app then tries to load even more modules, which fail due to the error above.

Later on in the logs, the error **Error: [WinError 206] The filename or extension is too long: 'C:\\Program Files\\Oculus\\Support\\oculus-runtime'.** also still appears. I tried uninstalling oculus runtime, and running remix again (on the 2024.1.1 version, while debugging and trying anything to make it work previously), but this did not fix the issue, the code just referenced other folder (i think it was a media codec or something).

I will try to provide any logs that might help, and I am looking forward to resolving the issue.

[kit_20240229_154242.log](https://github.com/NVIDIAGameWorks/rtx-remix/files/14449686/kit_20240229_154242.log)

**Comment by mbgamer05** [comment]
[kit_20240314_131713.log](https://github.com/NVIDIAGameWorks/rtx-remix/files/14596034/kit_20240314_131713.log)

noticied in this that
"py-spy.exe' exited with code 1"
normally that means it doesn't have a parameter which could be causing the crash but idk why

**Comment by 0ddly** [comment]
Uninstall "OpenCL, OpenGL, and Vulkan Compatibility Pack" which comes preinstalled on Windows 11 (unsure about Windows 10.) Recently learnt of this from someone on Discord, now just trying to spread the word as it seems to work with Laptops (the main systems affected by this.)

**Comment by MartinPihrtJR** [comment]
@0ddly thanks for a tip. I've tried to find the compability pack on my system, but nothing has shown up. Seems like I do not have it installed.

**Comment by NV-LL** [comment]
Hello! Will you please test this on the latest release and let us know if the issue persists? Thank you!

**Comment by MartinPihrtJR** [comment]
Hello @NV-LL . I am unfortunately on a traineeship in another country, with only my laptop. May I ask you what is the latest version of RTX Remix right now? I've tested on version 2024.2.1 and 2024.1.1. If there is a newer version, I will test the issue as soon as possible (approx end of month).

**Comment by NV-LL** [comment]
@MartinPihrtJR the Remix app is now on version 2024.3.0, available for update on the Omniverse launcher.

**Comment by MartinPihrtJR** [comment]
@NV-LL thanks for the info! I will take a look as soon as I get home, and update you with results.

**Comment by MartinPihrtJR** [comment]
# UPDATE
## Updated result
Hi @NV-LL, I've tested the problem on the latest version of RTX Remix. The results are below...
I've updated my RTX Remix to the latest version being **2024.3.1**. The app still does not work, with the same result as in the original issue. App launches a splashscreen, which disappears after a while and nothing more happens. I've included the crashlog from the kit itself.
## My view on the errors
Looking at the log myself, the errors seem to start after a read fail on line 2342 (not sure if related) shortly followed by the OSError: [WinError 87] The parameter is incorrect: 'C:' error leading to a cascade of more and more error forming as a result of unsuccessful loading of modules. Being just an assumption, only the developers at NVidia know how this thing works under the hood... :)
## Question
I want to completely uninstall RTX Remix & Omniverse to do a fully clean install (maybe some files from the previous versions were left somewhere causing the errors). Is there a guide for doing so?
# Files
[kit_20240530_211714.log](https://github.com/NVIDIAGameWorks/rtx-remix/files/15504999/kit_20240530_211714.log)

**Comment by NV-LL** [comment]
Hi @MartinPihrtJR! The Omniverse app has a [Launcher Cleanup Tool](https://docs.omniverse.nvidia.com/utilities/latest/cleanup-tool.html) that will remove all installed Omni apps. We recommend making a backup of all your data and projects first - the tool will allow you to keep your data, but backups help prevent a worst-case scenario. Then you can uninstall the launcher itself via Windows. Let us know if a clean reinstall helps!

**Comment by CodaCM** [comment]
I am having a similar issue where it crashes to the error remix::create d3d9" has failed. error code: 1.

**Comment by Infinite-Mouse** [comment]
I got an issue that looks like the situation in the video but maybe for a different reason.While starting the app,there's actually a window hidden behind the RTX remix loading box that says"Failed to create any gpu devices for compatibility mode." and I have no idea what to do with it.I am using a laptop with an Intel intergrated GPU and a RTX A3000 Laptop GPU(not compatible with Remix) and I plugged a RTX 3080 with 10GB VRAM outside.By the way the sample runs normally on my RTX 3080.Waiting for a fix.

**Comment by po-trottier** [comment]
Hey @MartinPihrtJR are you still crashing on startup ? Please test using the latest Toolkit release and let us know how it goes !

**Comment by MartinPihrtJR** [comment]
Hey @po-trottier the issue seems to be resolved. I have installed the latest version (1.4.2.1) using the NVIDIA app. Launching RTX Remix now successfully boots up to the main menu of the app. Honestly, I have long forgotten about this thread and gave up on trying to mod using the tool, but now I might try again. Thanks for the work on the tool, I hope it will evolve to support DirectX9.0c in the future (I have a game in mind I might want to mod that unfortunately runs on this API version).

**Comment by po-trottier** [comment]
If you browse through the community’s Discord server you might see some compatibility efforts there that may benefit your project :)

Closing the issue since it’s been resolved!
